pkg/sypgp/sypgp_test.go

```diff
@@ -197,6 +199,152 @@ func TestPushPubkey(t *testing.T) {
 	}
 }
 
+func TestEnsureDirPrivate(t *testing.T) {
+	tmpdir, err := ioutil.TempDir("", "test-ensure-dir-private")
```

Comment:
Could you please add `test.DropPrivilege(t)` and `test.ResetPrivilege(t)`? Thanks

Reply:
different PR